client/web/compose/src/views/Admin/Pages/Builder.vue

```diff
@@ -570,20 +595,38 @@ export default {
     },
 
     handleClone () {
+      const { namespaceID = NoID } = this.namespace
       let page = this.page.clone()
-      page = {
-        ...page,
-        pageID: NoID,
-        title: this.$t('copyOf', { title: this.page.title }),
-        handle: '',
-      }
 
-      const { namespaceID = NoID } = this.namespace
-      this.createPage({ namespaceID, ...page })
-        .then(({ pageID }) => {
-          this.$router.push({ name: 'admin.pages.builder', params: { pageID } })
+      this.loadPages({ namespaceID, force: true })
+        .then(pages => {
+          const pageTitles = pages.map(p => new compose.Page(p).title)
```

Comment:
Do we really need to map the pages to the class here? Look at the store, they already are